cleanreg.py

```diff
@@ -79,6 +82,13 @@ def parse_arguments():
     # check if keepimages is set that it is not negative
     if (args.keepimages is not None) and (args.keepimages < 0):
         parser.error("[-k] has to be a positive integer!")
+
+    # check if date is valid
+    if args.date is not None:
+        try:
+            datetime.strptime(args.date, '%d.%m.%Y')
```

Comment:
This should better be an ISO date format, maybe allow day (2018-05-31) and datetime (2018-05-31T18:03:24+00:00)